rj_gameplay/stp/role/__init__.py

```diff
@@ -58,6 +58,18 @@ def __call__(
         """
         ...
 
+    def unassigned_cost_fn(
+        self,
+        prev_result: Optional["RoleResult"],
+        world_state: rc.WorldState) -> float:
+        """Given the previous role assigment and current world state,
+        returns the cost of not assigning any robot/
```

Comment:
```suggestion
        returns the cost of not assigning any robot.
```
nit: typo
Thanks for adding this docstring!